apps/condo/domains/resident/schema/Resident.test.js

```diff
@@ -412,5 +424,32 @@ describe('Resident', () => {
             expect(objUpdated.updatedAt).toMatch(DATETIME_RE)
             expect(objUpdated.updatedAt).not.toEqual(objUpdated.createdAt)
         })
+
+        it('can be soft-deleted using update operation by current user with type resident', async () => {
```

Comment:
Try also to delete user while updating it

Reply:
Added extra checks in `737cbaa1`